apps/iiif/serializers/annotation.py

```diff
@@ -40,7 +40,7 @@ def get_dump_object(self, obj):
                 "resource": {
                     "@type": obj.resource_type,
                     "format": "text/html",
-                    "chars": obj.svg,
+                    "chars": obj.content,
```

Comment:
One of the challenges I think we're running into is that OCR-created annotations store HTML content in `svg`, while Readux-authored annotations store HTML content in `content`.  I switched line 43 to `obj.svg` in an earlier commit to get the fonts and styling of the HTML, but my change caused the human-created annotation contents to disappear.  

@jayvarner, what do you think would be involved in moving the OCR-generated HTML from the `svg` column to the `content` column in `annotations_annotation` and the sample data?

Reply:
That's good. It was not my intention to retain the svg label. Now seems like a good time to make the change.